orbitize/sampler.py

```diff
@@ -34,15 +36,17 @@ def __init__(self, system, like='chi2_lnlike', custom_lnlike=None):
             self.lnlike = getattr(orbitize.lnlike, like)
 
         self.custom_lnlike = custom_lnlike
+        self.use_c = use_c
```

Comment:
Not related to this PR, but what is the purpose of having `self.use_c` and `use_c` in the arguments of `run_sampler` and `_logl`. Some of the behavior has it as default `None` and some as default `True`. Is this an opportunity to override default values, and if so, do we actually implement it like that? Do we need all of them?